lib/15_in_words.rb

```diff
@@ -0,0 +1,90 @@
+require 'byebug'
+
+ONES = {
+    1 => "one",
```

Comment:
Typically you want symbols instead of strings when possible in stored constants like these hashes, since it has better memory usage.